qemu/tests/qemu_disk_img_convert.py

```diff
@@ -80,13 +81,20 @@ def run(test, params, env):
     if not md5:
         test.error("Fail to save tmp file")
     convert_test.destroy_vm()
-    n_params = convert_test.convert()
-    convert_test.compare_test(n_params)
-    convert_test.verify_info(n_params)
-    convert_test.start_vm(n_params)
 
-    # check md5sum after conversion
-    ret = convert_test.check_file(t_file, md5)
-    if not ret:
-        test.error("image content changed after convert")
-    convert_test.clean()
+    cluster_size_set = params.get("cluster_size_set")
+    if cluster_size_set == None:
```

Comment:
Refer to PEP8:  https://www.python.org/dev/peps/pep-0008/
```
Programming Recommendations

    Code should be written in a way that does not disadvantage other implementations of Python (PyPy, Jython, IronPython, Cython, Psyco, and such).

    For example, do not rely on CPython's efficient implementation of in-place string concatenation for statements in the form a += b or a = a + b. This optimization is fragile even in CPython (it only works for some types) and isn't present at all in implementations that don't use refcounting. In performance sensitive parts of the library, the ''.join() form should be used instead. This will ensure that concatenation occurs in linear time across various implementations.

    Comparisons to singletons like None should always be done with is or is not, never the equality operators.

    Also, beware of writing if x when you really mean if x is not None -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!

    Use is not operator rather than not ... is. While both expressions are functionally identical, the former is more readable and preferred.

    Yes:

    if foo is not None:

    No:

    if not foo is None:


```